management/status_checks.py

```diff
@@ -314,6 +314,20 @@ def run_domain_checks(rounded_time, env, output, pool):
 	for domain in sort_domains(ret, env):
 		ret[domain].playback(output)
 
+	run_custom_domain_checks(env, output)
+
+def run_custom_domain_checks(env, output):
+	header_pending = True
+	for qname, rtype, value in get_custom_dns_config(env):
+		if header_pending:
+			output.add_heading("Custom")
+			header_pending = False
+		result = query_dns(qname, rtype).replace('" "', '')
+		if value == result or '"' + value + '"' in result:
```

Comment:
If the return value is a single string, it won't be quoted.  If it's part of a list, it'll be surrounded in double quotes.

So, we match if either:
* value == result
* '"{value}"' in result

I suppose I could put that into a function to make it clearer.